views/dashboard/dashboard.hbs

```diff
@@ -56,6 +56,29 @@
             </div>
         </section>
 
+        {{#ifeq this.currentRole "Beta"}}
+            <section class="section section-my-material" data-step="8" data-intro="Hier finden Sie eine Übersicht über Ihre Materialien.">
+                <div class="container-fluid">
+                    <div class="row">
+                        <div class="col-sm-12"><h5>Meine Materialien</h5></div>
+                    </div>
+                    <div class="row tasks card-deck-row">
+                        {{#each myMaterials}}
+                            <div class="sc-card-wrapper col-xl-3 col-lg-4 col-md-6 col-sm-12">
+                                {{> 'lib/components/sc-card' link-text=../link-text}}
+                            </div>
+                        {{else}}
+                            <div class="col-md-8 offset-md-2 col-lg-12 offset-lg-0">
+                                <div class="alert">
+                                    Sie haben noch keine eigenen Materialien erstellt. Sie können sich aber auch Inspiration in Materialien Ihrer Kollegen holen. Schauen Sie doch im <a href="/my-material">"Meine Materialien"-Abschnitt</a> vorbei.
```

Comment:
gender the text?

Reply:
According to feministisch-sprachhandeln.org, the correct gender for "Kollegen" would be "Kollegx" (speak as "Kolleg-ix-es"). That took us a while to figure out...
Should I use "Kolleg*innen" instead or do we  as a Schul-Cloud have a guideline for that?